.github/dependabot.yml

```diff
@@ -14,8 +14,8 @@ updates:
       flink-dependencies:
         applies-to: security-updates
         patterns:
-          - "org.apache.flink:*"
+          - "*flink*"
 
       general-dependencies:
-        patterns:
-          - "*"
\ No newline at end of file
+        exclude-patterns:
+          - "*flink*"
```

Comment:
I think it would be safer to make sure they had the `org.apache` prefix but lets try this for now.